src/pages/index.js

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
This works for the `/` route but not for `/index.html`

We need to add an explicit redirect to `gatsby-node.js` to handle this:

```js
  // Without this, /index.html won't redirect to root.
  createRedirect({
    fromPath: '/index.html',
    redirectInBrowser: true,
    toPath: '/',
  });
```

Reply:
Good catch, thanks for this!